### PR TITLE
[155] Add Cocoapods

### PR DESCRIPTION
**Pull request by llleyelll**
Resolves https://github.com/nimblehq/ios-templates/issues/155

## What happened

- Setup `Podfile`
 
## Insight

- Update `Gemfile`
 
## Proof Of Work

Screen Shot 2021-08-26 at 11 57 37 AM

### CODE REVIEW

**Review by suho — APPROVED** [review]

**Comment by vnntsu** [comment]
@suho @llleyelll I think this PR doesn't depend on `feature/154-add-fastlane`. Please change the base to `develop` and merge it independently. What do you guy think?

**Comment by llleyelll** [comment]
> @suho @llleyelll I think this PR doesn't depend on `feature/154-add-fastlane`. Please change the base to `develop` and merge it independently. What do you guy think?

@vnntsu I set this PR to depend on `feature/154-add-fastlane` because I want to add `cocoapods` to `Gemfile` that was added from `feature/154-add-fastlane` branch. What do you think?

**Comment by suho** [comment]
@vnntsu Good point.
@llleyelll I think we can create change the base to `develop`, and you can create a Gemfile and add cocoapods into it. So we can merge it independently.

**Comment by llleyelll** [comment]
@suho @vnntsu I just rebased, please help check. Thank you 🙏

File: Podfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+target '{PROJECT_NAME}' do
+  # UI
+  pod 'SnapKit'
+  pod 'Kingfisher'
+
+  # Rx
+  pod 'RxSwift'
+  pod 'RxCocoa'
+  pod 'RxDataSources'
+  pod 'RxAlamofire'
```
Comment (minhnimble) [inline Podfile:21]: I think we can sort these pods alphabetically for each section as well? What do you think? 😁

**Review by minhnimble — COMMENTED** [review]
The rest LGTM 🚀

**Comment by suho** [comment]
@minhnimble Please help me to check this PR 😁

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**